client/src/pages/Chefs.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ChefSearch from "components/ChefSearch";
+import LoginForm from "components/LoginForm";
+import Navbar from "components/Navbar"
+
+const Chefs = ({ props }) => {
+  return (
+    <>
+      <Navbar/>
```

Comment:
Does the navbar need to be added in each component? This component should be in LoggedInComponent, so it shouldn't need to be here right?

Reply:
Yup, this needs to be taken out. I had that in there assuming that the user could search for chefs without being logged in.